README.md

```diff
@@ -387,10 +446,12 @@ We use nouns or adjectives for property names, not verbs or questions.
 
 ```php
 <?php
+declare(strict_types=1);
```

Comment:
Lets follow PSR-12 in our examples if our style guide says that Paysera style extens PSR-12